store/sqlstore/upgrade.go

```diff
@@ -743,5 +732,17 @@ func UpgradeDatabaseToVersion516(sqlStore SqlStore) {
 			sqlStore.GetMaster().Exec("ALTER TABLE Tokens MODIFY Extra text")
 		}
 		saveSchemaVersion(sqlStore, VERSION_5_16_0)
+
+		// Fix mismatches between the canonical and migrated schemas.
+		sqlStore.AlterColumnTypeIfExists("TeamMembers", "SchemeGuest", "tinyint(4)", "boolean")
```

Comment:
If a customer /did/ upgrade through the `UpgradeDatabaseToVersion514`, it's safe to run these a second time, as they are idempotent.